### PR TITLE
Fixed backward compatibility for GKE e2e test

### PR DESCRIPTION
**Pull request by mangchiandjjoe**

### CODE REVIEW

File: test/bookstore/gke/nginx.conf [hunk 1]
```diff
@@ -42,7 +42,7 @@ http {
       # Begin Endpoints v2 Support
       endpoints {
         on;
-        server_config /etc/nginx/endpoints/server_config.pb.txt;
+        server_config /etc/nginx/server_config.pb.txt;
```
Comment (qiwzhang) [inline test/bookstore/gke/nginx.conf]: could you also add

api /etc/nginx/endpoints/service.json

back.  I bet customer custom nginx config have that one too.   We want to test it to make sure it works.

Comment (qiwzhang) [inline test/bookstore/gke/nginx.conf]: Just revert this change:  https://github.com/cloudendpoints/esp/commit/2f75b9819b9d78e8235bdef2af9ab7ee72fc79c6

Comment (mangchiandjjoe) [inline test/bookstore/gke/nginx.conf]: Done

**Comment by mangchiandjjoe** [comment]
e2e tests against the PR was succeeded.

https://endpoints-jenkins.appspot.com/job/jenkins-testing/job/pr-e2e/59/
